srcpkgs/ffmpeg/template

```diff
@@ -183,3 +184,10 @@ ffmpeg-devel_package() {
 		vmove usr/share/man/man3
 	}
 }
+ffplay_package() {
+	short_desc+="FFmpeg video resampling library"
```

Comment:
shortdesc seems wrong, this isn't the library...

Reply:
oops, good catch